src/Latestdocument.jsx

```diff
@@ -0,0 +1,59 @@
+import React, {useState, useEffect} from 'react'
+import ListItem from '@mui/material/ListItem';
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography'; 
+import Link from '@mui/material/Link';
+import ListItemText from '@mui/material/ListItemText';
+
+
+
+const Latestdocument = () => { 
+     const [documentos, setDocumentos] = useState([])
+      
+
+     const URL = 'https://jsonplaceholder.typicode.com/posts'
+     
+     const showData = async () => { 
+
+        const response = await fetch(URL)
+        const data = await response.json() 
+        setDocumentos(data)
+        
```

Comment:
This is a bad practice; you should put this request in the async folder and use the https://github.com/Ditmar/plagiarist-g3/blob/master/src/async/api.js  file to make this request.